wow-apps/symfony-slack-bot/3.2/config/packages/wow_apps_slack.yaml

```diff
@@ -0,0 +1,12 @@
+# SlackBot Configuration
+# Documentation https://github.com/wow-apps/symfony-slack-bot/wiki
+wow_apps_slack:
+    api_url: ''
+    default_icon: 'http://cdn.wow-apps.pro/slackbot/slack-bot-icon-48.png'
+    default_channel: 'general'
+    quote_color:
+        default: '#607D8B'
+        info: '#2196F3'
+        warning: '#FF5722'
+        success: '#8BC34A'
+        danger: '#F44336'
```

Comment:
Repeating all the default config does not look sensible to me. When the bundle has good defaults, the recipe should rely on them instead of copy-pasting them.

Reply:
@stof 
You mean, that if I have default values in [configuration file](https://github.com/wow-apps/symfony-slack-bot/blob/master/DependencyInjection/Configuration.php) I don't need to copy in recipe? Or I don't understand :(

Reply:
yup. If it's default, it's the one used if no value given.

You should keep the minimal required in the config file.

Reply:
You should remove all the `quote_color` config

Reply:
@Nyholm @Taluu 
Thanks for explanations